api/services/elastigroup/aws/schemas/items-status-f.yaml

```diff
@@ -40,3 +40,11 @@ properties:
     type: string
     title: Status
     example: fulfilled
+  lockStatus:
+    type: strinrg
```

Comment:
You have a typo here.
Also, this should be enum, right? (with 'locked'/'unlocked' as allowed values)